Source/WebKit/InspectorGResources.cmake

```diff
@@ -9,13 +9,13 @@ macro(WEBKIT_BUILD_INSPECTOR_GRESOURCES _derived_sources_dir _output_file)
     )
 
     GLIB_COMPILE_RESOURCES(
-        OUTPUT        ${_derived_sources_dir}/${_output_file}
+        OUTPUT        ${_output_file}
         SOURCE_XML    ${_derived_sources_dir}/InspectorGResourceBundle.xml
         RESOURCE_DIRS ${_derived_sources_dir}/InspectorResources/WebInspectorUI
     )
 
-    add_custom_target(${_output_file}
+    add_custom_target(InspectorResources
```

Comment:
For future reference: Making the name fixed here prevents using the macro more than once for each port; but this is fine because that's exactly how it's used at the moment.